client-admin/.dockerignore

```diff
@@ -0,0 +1,42 @@
+### SublimeText ###
+*.sublime-workspace
+
+### OSX ###
+.DS_Store
+.AppleDouble
+.LSOverride
+Icon
+
+# Thumbnails
+._*
+
+# Files that might appear on external disk
+.Spotlight-V100
+.Trashes
+
+### Windows ###
+# Windows image file caches
+Thumbs.db
+ehthumbs.db
+
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# App specific
+build
+dist
+coverage
+node_modules
+bower_components
+.tmp
+lib
+npm-debug.log
+.polis_s3_creds_client.json
+.env_dev
+polis.config.js
```

Comment:
```suggestion
```

Can we leave this off? This would allow a custom `polis.config.js` to be brought in via `ADD . .` if it exists

Reply:
The Dockerfile currently copies the polis.config.template.js file to polis.config.js inside of the container, so any customization could be done to the polis.config.template.js.
Any changes to polis.config.js would actaully be clobbered by that Dockerfile action.
I'm open to just doing away with the template file, and copying the polis.config.js file directly.

Reply:
Hm. It's not explicitly documented as such, but the order is:

```
COPY polis.config.template.js polis.config.js
COPY . .
```
And so polis.config.js can be created locally, and it's gitignored so won't be committed or make the git workspace dirty, but gets copied over the template during second COPY (so long as it's not in dockerignore).

I'd be down to document that as the blessed way to customize build config

Reply:
I might be mistaken, but it looks like if you have both polis.config.js and polis.config.template.js, then first the template gets copied to polis.config.js, and then gets clobbered by the other polis.config.js (if not docker ignored). That seems clumsy. But I guess it provides a default if the user has not specified a polis.config.js file on their own. Hmmm... 🤔

Reply:
For now I will change it back to how I found it. ie remove polis.config.js from .dockerignore

Reply:
kthx! yes, that's exactly how i was using it (i'm ok to ditch if harmful, but thx for keeping for now)